assets/src/components/detours/diversionPanel.tsx

```diff
@@ -74,7 +74,12 @@ export const DiversionPanel = ({
 
         {missedStops && (
           <section className="pb-3">
-            <h2 className="c-diversion-panel__h3">Missed Stops</h2>
+            <h2 className="c-diversion-panel__h2">
+              Missed Stops{" "}
+              <Badge bg="missed-stop" className="c-diversion-panel__h3">
```

Comment:
**praise**: I love the way that this ended up being usable via the `bg` parameter!